sdk/openai/openai/vitest.browser.config.ts

```diff
@@ -1,10 +1,12 @@
-
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
 import { defineConfig, mergeConfig } from "vitest/config";
 import viteConfig from "../../../vitest.browser.shared.config.ts";
-import { resolve } from "node:path";
+import { fileURLToPath } from "url";
+import { dirname, resolve } from "path";
+
+const __dirname = dirname((import.meta.url);
```

Comment:
This applies here too: https://github.com/Azure/azure-sdk-for-js/pull/33264/files#r1977963365

Reply:
Fixed